components/devRelEvents/DevRelEvent.tsx

```diff
@@ -0,0 +1,149 @@
+import {
+  AccordionButton,
+  AccordionItem,
+  AccordionPanel,
+  Flex,
+  Icon,
+} from "@chakra-ui/react";
+import { useTrack } from "hooks/analytics/useTrack";
+import { FaMapMarkerAlt } from "react-icons/fa";
+import { FiCalendar, FiClock } from "react-icons/fi";
+import { Badge, Heading, LinkButton, Text } from "tw-components";
+
+interface EventProps {
+  type: string;
+  title: string;
+  timestamp: string;
+  location: string;
+  description: string;
+  link: string;
+}
+
+export const DevRelEvent: React.FC<EventProps> = ({
+  type,
+  title,
+  timestamp,
+  location,
+  description,
+  link,
+}) => {
+  const trackEvent = useTrack();
+
+  return (
+    <AccordionItem
```

Comment:
it's not clear that this is clickable, if anything let's make the entire card clickable (accordionbutton can have the padding instead of the item and then everything will be clickable)